tests/playground.html

```diff
@@ -178,6 +179,65 @@
   }
 }
 
+function fakeDrag(id, dx, dy, opt_workspace) {
+  var ws = opt_workspace ? opt_workspace : Blockly.getMainWorkspace();
+  var blockToDrag = ws.getBlockById(id);
+
+  if (!blockToDrag) {
+    return;
+  }
+  var blockTop = blockToDrag.svgGroup_.getBoundingClientRect().top;
+  var blockLeft = blockToDrag.svgGroup_.getBoundingClientRect().left;
+
+  // Click somewhere on the block.
+  var mouseDownEvent = new MouseEvent('mousedown',
+      {clientX: blockLeft + 5, clientY: blockTop + 5});
+  blockToDrag.onMouseDown_(mouseDownEvent);
+
+  // Throw in a move for good measure.
+  window.setTimeout(
```

Comment:
why the nested settimeouts? Is it to have a visible delay between the mousedown, move and up?